desktop/cmp/contextmenu/ContextMenu.js

```diff
@@ -17,8 +17,9 @@ import {ContextMenuItem} from './ContextMenuItem';
 /**
  * ContextMenu
  *
- * To add a Context Menu to a component, specify a renderContextMenu() method on the
- * component that returns this object.
+ * Not typically used directly by applications.  To add a Context Menu to a
+ * component, decorate the component with ContextMenuSupport and implement
+ * getContextMenuItems().
```

Comment:
As we have one `@see` annotation on this doc comment already, might make sense to convert this note into a pointer to the decorator class.